src/main/webapp/mentors.js

```diff
@@ -1,3 +1,6 @@
+import standardizeFetchErrors from '/js/fetch_handler.js';
+getMentors();
```

Comment:
Moving `getMentors()` from onload to here changes the behavior of the code, so make sure it's intended. This function will now run when the script is loaded in the HTML (before the body is loaded) instead of after the body is loaded.